src/pages/api/index.md

```diff
@@ -1,5 +1,5 @@
 ---
 title: Photoshop API Reference
 description: This is the OpenAPI page of Adobe Photoshop Service
-openAPISpec: https://raw.githubusercontent.com/AdobeDocs/cis-photoshop-api-docs/main/static/swagger.json
+openAPISpec: /swagger.json
```

Comment:
Hi Sudipta, I only see this file change, but not the actual text change in remove background API doc. Is it expected?